Simplified.xcodeproj/project.pbxproj

```diff
@@ -6076,7 +6133,7 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Simplified/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				CURRENT_PROJECT_VERSION = 9;
+				CURRENT_PROJECT_VERSION = 13;
```

Comment:
Is there a reason this is 13 but the others are 10?

Reply:
I think it's because that's the build number of the R2dev target, which is the one I was using. In my next PR where I integrate the changes for r2-navigator and create an actual build, I will straighten those up.